src/Compilers/CSharp/Test/CommandLine/CommandLineTests.cs

```diff
@@ -1104,6 +1104,12 @@ public void ArgumentParsing()
             Assert.True(parsedArgs.DisplayHelp);
             Assert.False(parsedArgs.SourceFiles.Any());
 
+            parsedArgs = CSharpCommandLineParser.ScriptRunner.Parse(new[] { "/version" }, _baseDirectory, s_defaultSdkDirectory);
```

Comment:
By 'verifies' do you mean that the compiler doesn't crash? I'm not sure what other purpose such tests would serve.